packages/notification-center/src/components/notification-center/components/NotificationsList.tsx

```diff
@@ -47,4 +47,5 @@ export function NotificationsList({
 
 const notificationsListCss = css`
   height: 400px;
+  overflow-y: auto;
```

Comment:
the fix... the `InfiniteScroll` component has a prop `scrollableTarget="notifications-list"` which points to the parent element and it should be a scrollable element otherwise the infinite scroll won't work